src/components/about/officer-profile-list.svelte

```diff
@@ -13,7 +13,7 @@
   };
 
   const formattedTerms = TERMS.map(formatTerm);
-  let currentTermIndex = TERMS.length - 1;
+  let currentTermIndex = TERMS.length - 2;
```

Comment:
Set this to 0.

Reply:
setting it to 0 doesn't make the list appear until i select spring 2021, then fall 2021

![image](https://user-images.githubusercontent.com/54593375/134606543-be87595f-a1e2-4c43-bb5a-c435375adb2f.png)